istioctl/pkg/authz/listener.go

```diff
@@ -125,7 +127,15 @@ func ParseListener(listener *v2.Listener) *ParsedListener {
 	}
 
 	for _, fc := range listener.FilterChains {
-		parsedFC := &filterChain{tlsContext: fc.TlsContext}
+		tlsContext := &auth.DownstreamTlsContext{}
+		if fc.TransportSocket != nil && fc.TransportSocket.Name == util.EnvoyTLSSocketName {
+			if err := ptypes.UnmarshalAny(fc.TransportSocket.GetTypedConfig(), tlsContext); err != nil {
+				continue
```

Comment:
log error when this happens.

Reply:
although this should never happen correct?

Reply:
This should never happen. I wasn't sure if we log errors in istioctl code?

Reply:
miss this was in istioctl.